aspnetcore/signalr/configuration.md

```diff
@@ -177,6 +177,8 @@ Log levels available to the JavaScript client are listed below. Setting the log
 | `Debug` | Diagnostic messages useful for debugging. |
 | `Trace` | Very detailed diagnostic messages designed for diagnosing specific issues. |
 
+For details on how to enable logging in the Java client, see [Logging in SignalR](xref:signalr/java-client#Add-logging).
```

Comment:
Probably not in scope for this PR, but this sentence is kind of hidden, we might want to rework the logging stuff at some point

Reply:
I'm just gonna copy over the logging instructions from the other page. Better to just have the info here to match the other sections.